src/clients/SpokePoolClient.ts

```diff
@@ -784,11 +784,8 @@ export class SpokePoolClient extends BaseAbstractClient {
       }
     }
 
-    if (eventsToQuery.includes("FilledRelay") || eventsToQuery.includes("FilledV3Relay")) {
-      const fillEvents = [
-        ...((queryResults[eventsToQuery.indexOf("FilledRelay")] ?? []) as FilledRelayEvent[]),
-        ...((queryResults[eventsToQuery.indexOf("FilledV3Relay")] ?? []) as FilledV3RelayEvent[]),
-      ];
+    if (eventsToQuery.includes("FilledV3Relay")) {
+      const fillEvents = [...((queryResults[eventsToQuery.indexOf("FilledV3Relay")] ?? []) as FilledV3RelayEvent[])];
```

Comment:
Would something like this give us any tsc errors? I'm not sure the spread operator is necessary here

```suggestion
      const fillEvents: FilledV3RelayEvent[] = queryResults[eventsToQuery.indexOf("FilledV3Relay")] ?? [];
```

Reply:
tsc isn't happy about this because `queryResults` is an ethers Event, whereas `FilledV3RelayEvent` is a more tightly-scoped type.